ext/node/polyfills/util.ts

```diff
@@ -318,6 +321,64 @@ export async function aborted(
   return abortPromise.promise;
 }
 
+function prepareStackTrace(_error, stackTraces) {
+  return ArrayPrototypeMap(stackTraces, (stack) => {
+    return ({
+      functionName: stack.getFunctionName() ?? "",
+      // TODO(kt3k): This needs to be script's id
+      scriptId: "0",
+      scriptName: stack.getFileName(),
+      lineNumber: stack.getLineNumber(),
+      column: stack.getColumnNumber(),
+      columnNumber: stack.getColumnNumber(),
+    });
+  });
+}
+
+const kDefaultMaxCallStackSizeToCapture = 200;
+
+// deno-lint-ignore-start
+/**
+ * Returns the call sites of the current call stack
+ * @param frameCount The limit of the number of frames to return
+ * @param _options The options
+ * @returns The call sites
+ */
+export function getCallSites(
+  frameCount = 10,
+  options: unknown = { __proto__: null },
+) {
+  validateNumber(
+    frameCount,
+    "frameCount",
+    0,
+    kDefaultMaxCallStackSizeToCapture,
+  );
+  if (options) {
+    validateObject(options, "options");
+  }
+  const target = {};
+  // deno-lint-ignore prefer-primordials
+  const original = Error.prepareStackTrace;
+  // deno-lint-ignore prefer-primordials
+  const limitOriginal = Error.stackTraceLimit;
+
+  // deno-lint-ignore prefer-primordials
+  Error.stackTraceLimit = frameCount;
+  // deno-lint-ignore prefer-primordials
+  Error.prepareStackTrace = prepareStackTrace;
+  // deno-lint-ignore prefer-primordials
+  Error.captureStackTrace(target, getCallSites);
```

Comment:
`ErrorCaptureStackTrace` should be in primordials.

https://github.com/denoland/deno/blob/3849ace1d6b3f184645aaf077539ef7ef12f560c/ext/console/01_console.js#L64